snapshot_dbg_cli/delete_snapshots_command.py

```diff
@@ -127,9 +127,11 @@ def cmd(self, args, cli_services):
         user_output.error('Delete aborted.')
         return
 
-    debugger_rtdb_service.delete_breakpoints(args.debuggee_id, snapshots)
+      debugger_rtdb_service.delete_breakpoints(args.debuggee_id, snapshots)
```

Comment:
I suppose that if there's one criticism of Python, this is it

Reply:
Indeed, but yes, this was intentional :)